iceberg-catalog-migrator/gradle/libs.versions.toml

```diff
@@ -75,6 +76,7 @@ logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "lo
 logcaptor = { module = "io.github.hakky54:logcaptor", version.ref = "logcaptor" }
 picocli = { module = "info.picocli:picocli", version.ref = "picocli" }
 slf4j = { module = "org.slf4j:log4j-over-slf4j", version.ref = "slf4j" }
+testcontainers-bom = { module = "org.testcontainers:testcontainers-bom", version.ref = "testcontainers" }
```

Comment:
Is it used?

Reply:
testcontainer dependency needed as 

`PolarisContainer extends GenericContainer<PolarisContainer> {`

So, added a dependency in https://github.com/apache/polaris-tools/pull/32/files#diff-63fbf4e3b782dbcabcab9699b864aea9b96668ec6215bc8951f8cb6ba2223e66R131

Did you mean why depend on bom instead of a specific dependency version?

Reply:
Aah. Done. Removed.